plugins/scorecards/README.md

```diff
@@ -0,0 +1,52 @@
+# Scorecards
+
+This plugin represents scorecards which are rendered as a part of the default Backstage Tech Insights feature.
+It provides UI for the scorecards as well as a check overview.
+
+## Installation
+
+### Install the plugin
+
+```bash
+# From your Backstage root directory
+cd packages/app
+yarn add @backstage/plugin-scorecards
+```
+
+### Adding the plugin to your `packages/app`
+
+```tsx
+// In packages/app/src/App.tsx
+
+import { EntityScorecardContent } from '@backstage/plugin-scorecards';
+
+<Route path="/scorecards" element={<EntityScorecardContent />} />;
```

Comment:
hm! should this really have a route like this? it's a bit unexpected to see the same thing used both as a standalone page and as an entity tab (which requires the namespace/kind/name triplet in the route)

Reply:
Yeah, I will remove this as tbh I see no value in having it as a standalone page either way.